### ISSUE TITLE
Supplying doc_id when inserting

### ISSUE THREAD

**Issue by mkohram-quotient** [msiemens/tinydb#345]
Hi!

Looking through advanced usage docs I see that I can specify a doc_id when inserting data:
```python
>>> db.insert(Document({'name': 'John', 'age': 22}, doc_id=12))
12
```

When I try to do this, it seems like my doc_id is getting ignored:
```python
>>> from tinydb.table import Document
>>> from tinydb import TinyDB
>>> db = TinyDB('test.json')
>>> doc = Document({'data': 'some_value'}, doc_id=42)
>>> db.insert(doc)
1
```

Am I missing something?

**Comment by eugene-eeo** [comment]
Which version of TinyDB are you using?

**Comment by selimt** [comment]
It seems like the master branch has this fix but the 4.1.1 tag doesn't. Would it be possible to release a new version?
Thanks.

**Comment by eugene-eeo** [comment]
cc @msiemens

**Comment by msiemens** [comment]
Oh, seems like I wanted to wait a couple of days if there are more changes before releasing 5040649 but then forgot to release a new version. Anyways, this should be fixed in v4.2.0 🙃

**Comment by msiemens** [comment]
Closing due to inactivity.
